iceoryx_posh/source/roudi/roudi_cmd_line_parser.cpp

```diff
@@ -238,5 +238,29 @@ units::Duration CmdLineParser::getProcessKillDelay() const noexcept
     return m_processKillDelay;
 }
 
+iox::log::LogStream& operator<<(iox::log::LogStream& logstream, const MonitoringMode& mode)
```

Comment:
According to codecov this is not tested. Could you please write a little test for this function.

Reply:
Just realised there is not `CmdLineParser` test so far. I remember @budrus didn't want to have issues created for these kind of findings. I can increase the test coverage of RouDi in a follow-up. If you look [here](https://codecov.io/gh/eclipse/iceoryx?branch=master), that's our main weak spot.

@budrus Ok from your side to create an issue on this? Happy to work on it after the 0.9 release